jobbergate-api/tests/apps/job_script_templates/test_routers.py

```diff
@@ -679,6 +680,106 @@ async def test_upsert_new_file(
         assert template_file["filename"] == dummy_file_path.name
         assert template_file["file_type"] == file_type
 
+    @pytest.mark.parametrize(
+        "is_owner, permissions",
+        [
+            (True, [Permissions.JOB_TEMPLATES_CREATE]),
+            (False, [Permissions.ADMIN]),
+            (False, [Permissions.JOB_TEMPLATES_CREATE, Permissions.MAINTAINER]),
+        ],
+    )
+    async def test_upsert_new_file_by_url(
```

Comment:
I think that this test and `test_upsert_new_file_by_url` should verify for all supported schemes, i.e. `http`, `https` and `s3` in this case.

Reply:
That's a fair point.

Reply:
Addressed in 370c6dd